### PR TITLE
Add offline jail duration config to genesis

### LINKED ISSUE #353: Add support for new valset fields
After v0.9.0 of contracts, valset has two new fields in the instantiation message:

- VerifyValidators bool `json:"verify_validators"`
- OfflineJailDuration uint64 `json:"offline_jail_duration"

Add these fields to valset init during bootstrap (after #351).

Sensible values could be `VerifyValidators = true`, that is, enable validators verification, and `OfflineJailDuration = 86400` (one day jailing period for being offline).

### PR DESCRIPTION
**Pull request by alpe**
Resolves #353

### CODE REVIEW

**Comment by alpe** [comment]
I am looking at the failing test

**Comment by alpe** [comment]
Blocked by https://github.com/confio/poe-contracts/issues/146

File: x/poe/types/genesis.go [hunk 1]
```diff
@@ -254,12 +255,18 @@ func (c *EngagementContractConfig) ValidateBasic() error {
 
 // ValidateBasic ensure basic constraints
 func (c ValsetContractConfig) ValidateBasic() error {
+	if c.VerifyValidators {
+		return ErrInvalid.Wrap("not supported currently. See https://github.com/confio/tgrade/issues/389")
```
Comment (maurolacy) [inline x/poe/types/genesis.go:259]: 👍🏼

**Review by maurolacy — APPROVED** [review]
LGTM.

**Pull request merged**